src/main.c

```diff
@@ -2247,6 +2361,22 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
         }
     }
     if (!g->typing) {
+        if (key == CRAFT_KEY_VIEW_KEYBINDS)
+        {
+            parseConfigToPrintKeybinds();
+        }
+        if (key == CRAFT_KEY_INCREMENT_DPI)
+        {
+            incrementDPI();
+        }
+        if (key == CRAFT_KEY_DECREMENT_DPI)
+        {
+            decrementDPI();
+        }
+        if (key == CRAFT_KEY_SET_DEFAULT_DPI)
+        {
+            setDefaultDPI();
+        }
```

Comment:
Instead of using multiple if statements, try using else if statements as it will decrease the complexity of the code.